### ISSUE TITLE
Is the model prediction probability calibrated?

### ISSUE THREAD

**Issue by Vedant-R** [tensorflow/decision-forests#71]
I understand that the sklearn random forest models prediction probabilities are not calibrated and we need to add steps in between to calibrate it.

Just wanted to understand if the prediction probabilities of tensorflow decision forests are calibrated?

If not, how can be calibrate it?

**Comment by janpfeifer** [comment]
No, they are not. But not by much usually -- depending on your use case it may not matter.

The extended discussion in [TF Forum here](https://discuss.tensorflow.org/t/is-the-tensoflow-decision-forests-prediction-probability-calibrated/6388), for others following up on this.

Leaving this issue as an enhancement to remind us to add automatic calibration option.
